cmd/exec/operations.go

```diff
@@ -439,6 +440,82 @@ func executeWorkflow(url string) (executeResponse, error) {
 
 }
 
+func executeEvent(url string) (string, error) {
+
+	// Read input data from flag file
+	inputDataOsFile, err := os.Open(localAbsPath)
+
+	if err != nil {
+		return "", err
+	}
+
+	defer inputDataOsFile.Close()
+	byteResult, _ := ioutil.ReadAll(inputDataOsFile)
+	var event map[string]interface{}
+	json.Unmarshal([]byte(byteResult), &event)
+
+	// Read input data as bytes.buffer
+	inputData, err := safeLoadFile(localAbsPath)
+
+	if err != nil {
+		log.Fatalf("Failed to load input file: %v", err)
+	}
+
+	if inputData.Len() == 0 {
+		return event["id"].(string), errors.New("empty file ")
```

Comment:
In most cases, I think it's easier and better to return the null result for the main return value if an error occurs. This isn't a strict rule. In this case, I can see that the caller will use the returned event ID as part of the error message. That makes sense. 

However, in this case, it can result in an uncatchable error. The `event["id"].(string)` can crash in multiple ways. Specifically, if `id` is undefined, or if it was unmarshaled as a non-string type.